packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -587,17 +676,33 @@ class BackdropFilter extends SingleChildRenderObjectWidget {
   /// type for performance reasons.
   final bool enabled;
 
+  /// The backdrop key that identifies the backdrop this filter will apply to.
+  ///
+  /// The default value for the backdrop key is `null`.
+  final BackdropKey? backdropKey;
+
+  /// Whether to look up the [backdropKey] from a parent [BackdropGroup].
+  final bool useSharedKey;
```

Comment:
I think this should be private, `_useSharedKey`.  People shouldn't be latching behavior based on this internal detail.

Reply:
Yeah that sounds good to me. I made both members private